app/app/static/custom/queue.js

```diff
@@ -155,34 +155,82 @@ const terminateSession = async(data) => {
 function validateUserInput(data) {
 	// Check student name
 	if (data.studentName.length > 50) {
-		alert("'Student Name' field should is too long.");
+		Swal.fire({
+			toast: true,
+			position: 'top-end',
+			showConfirmButton: false,
+			timer: 3000,
+			timerProgressBar: true,
+			padding: '1em',
```

Comment:
code repetition, consider using a constant that defines

```js
const DEFAULT_SWAL_PROPS ={
toast: true,
			position: 'top-end',
			showConfirmButton: false,
			timer: 3000,
			timerProgressBar: true,
			padding: '1em',
			}
```
then you can reuse this by doing the following
```js
Swal.fire({
...DEFAULT_SWAL_PROPS
text: "'Student Name' field should is too long."
}
```

Reply:
or create a custom function that only takes 1 parameter - text